serverConnection/startserver.go

```diff
@@ -0,0 +1,18 @@
+package serverConnection
```

Comment:
Remove this dir if not required.

Reply:
this is mostly required directory.... without this i can not start server.
before i put this in connection package where this cause looping of packages, i asked u than u said make new dir